README.md

```diff
@@ -6,17 +6,17 @@ This repository maintains the templates used in conjunction with the [VS Code To
 
 > ℹ️ **Toradex Supported Templates** are templates with development stacks commonly used by customers of the Torizon platform and are maintained by the Torizon team at Toradex.
 
-> ⚠️ Support for **x86-64** and **riscv64** architectures are experimental.
+> ⚠️ Support for **x86-64** architectures are experimental.
 
 | TEMPLATE | DESCRIPTION | RUNTIME | LANGUAGE | HW ARCH | FOLDER |
 | -------- | ----------- | ------- | -------- | ------- | ------ |
-| ![](https://raw.githubusercontent.com/toradex/vscode-torizon-templates-documentation/refs/heads/main/thumbnails/cppconsole.png?raw=true) | C++ Makefile Console Application | libc | C++ | ![](assets/img/arm32.png?raw=true&id=2) ![](assets/img/arm64.png?raw=true&id=2) ![](assets/img/riscv64.png?raw=true&id=2) ![](assets/img/x8664.png?raw=true&id=2) | [cppConsole](./cppConsole) |
+| ![](https://raw.githubusercontent.com/toradex/vscode-torizon-templates-documentation/refs/heads/main/thumbnails/cppconsole.png?raw=true) | C++ Makefile Console Application | libc | C++ | ![](assets/img/arm32.png?raw=true&id=2) ![](assets/img/arm64.png?raw=true&id=2) ![](assets/img/x8664.png?raw=true&id=2) | [cppConsole](./cppConsole) |
```

Comment:
Also the `![](assets/img/x8664.png?raw=true&id=2)` needs to be removed from the templates where it has been removed.

Reply:
This should be fixed on 7258c62